pytorch_lightning/trainer/connectors/logger_connector/epoch_result_store.py

```diff
@@ -281,18 +261,23 @@ def __getitem__(self, key: str) -> Any:
         return self._internals.get(key, None)
 
     @property
-    def has_split_and_opt_idx(self):
-        """
-        This function informs if we are running within training batch loop
-        """
-        return self._split_idx is not None and self._opt_idx is not None
-
-    @property
-    def extra_info(self):
+    def info(self):
```

Comment:
should this return a dataclass? right now there isn't much structure as to what's returned in this map which could lead to breakages later if keys are renamed

Reply:
This property is only called once and within our code. It's more like an utility so In don't see the point of going the extra mile and adding a dataclass.

But we could